services/core-api/app/api/mines/tailings/models/tailings.py

```diff
@@ -63,11 +63,25 @@ class MineTailingsStorageFacility(AuditMixin, Base):
         'nullsfirst(desc(MinePartyAppointment.start_date)), nullsfirst(desc(MinePartyAppointment.end_date))'
     )
 
+    qualified_persons = db.relationship(
+        'MinePartyAppointment',
+        lazy='select',
+        primaryjoin=
+        'and_(MinePartyAppointment.mine_tailings_storage_facility_guid == MineTailingsStorageFacility.mine_tailings_storage_facility_guid, MinePartyAppointment.mine_party_appt_type_code == "TQP", MinePartyAppointment.deleted_ind == False)',
+        order_by=
+        'nullsfirst(desc(MinePartyAppointment.start_date)), nullsfirst(desc(MinePartyAppointment.end_date))'
+    )
+
     @hybrid_property
     def engineer_of_record(self):
         if self.engineer_of_records:
             return self.engineer_of_records[0]
 
+    @hybrid_property
+    def qualified_person(self):
+        if self.qualified_persons:
+            return self.qualified_persons[0]
```

Comment:
Are we only allowing a single qualified person?  I thought there could be more than one, but I may be wrong about that.

Reply:
Currently, a mine can have multiple Qualified Persons since they are not tied to a TSF at all, but my understanding is that a TSF will only have one QP going forward